### ISSUE TITLE
Minor edit to newsletter confirmation page

### ISSUE THREAD

**Issue by ashlchap** [reskillamericans/website#156]
Here's the page copy I'm referring to: 
https://reskillamericans.us7.list-manage.com/subscribe/post?u=e275039f79d674f29f0ab6f3b&id=df5e7498de

I'd like to replace it with the below: (I simply deleted our mailing address and the repeated thank you at the top)
******
Thanks for subscribing to Reskill Americans!

Your subscription to our newsletter has been confirmed.

We will be keeping you informed about the latest news from our program (about one email per month).

You can always find out the latest by checking out these sites:

The Reskill Americans News Page (keep same link as currently)
Reskill Americans YouTube Channel keep same link as currently)

Feel free to contact us any time at info@reskillamericans.org.

**Comment by mckoss** [comment]
The page you're linking to is NOT a confirmation page.  It is MailChimp's subscription page.  So - the user has not yet subscribed when they see this.

![image](https://user-images.githubusercontent.com/82750/152094777-6f72cdab-52d8-412e-8967-d49e84af6694.png)

AFTER they sign up - they get the text you're referencing (I thought it was an email - but I guess you're seeing it at the end of the sign-up flow).

**Comment by mckoss** [comment]
If they type into the email field on our page - it just get's carried along to pre-fill the first line of this form - which is the actual subscription page.

**Comment by ashlchap** [comment]
Yes, I had seen the confirmation page when I hit the arrow at first, but I'm no longer seeing that copy anywhere (which I originally posted in this thread). It's not even being emailed to me. Anyway, I am fine with the subscription page now, but I had recommended (in what's app to you) VERY TINY tweaks, to replace those first sentences with:
 Thanks for joining our mailing list! You are now added to our monthly newsletter which will share updates about our program and our participants.

(Again, not critical vs other stuff)

**Comment by mckoss** [comment]
But they are not already added to the newsletter at this point.  They have to hit Subscribe, so I don't think that sentence can say they are "now added" since they will think they are done.
@ashlchap

**Comment by ashlchap** [comment]
But we already are thanking them for joining the mailing list? Ok, How about: 
Thanks for your interest in Reskill Americans! By joining our mailing list, you will receive our monthly newsletter and stay updated about our program and our participants.
